rmw_implementation/package.xml

```diff
@@ -4,7 +4,7 @@
   <name>rmw_implementation</name>
   <version>2.6.0</version>
   <description>Proxy implementation of the ROS 2 Middleware Interface.</description>
-  <maintainer email="alejandro@openrobotics.org">Alejandro Hernandez Cordero</maintainer>
```

Comment:
Let's make sure to convert old maintainers to authors to recognize their contribution.

(same below)

Reply:
Ah, I didn't know to do this - will do!

Reply:
Is this true for `setup.py` files as well?